packages/dom/src/clone-dom.js

```diff
@@ -26,6 +26,12 @@ export function cloneNodeAndShadow({ dom, disableShadowDOM }) {
 
     parent.appendChild(clone);
 
+    // shallow clone should not contain children
+    // these might get added due to DOM side effects
```

Comment:
```suggestion
    // these might get added due to DOM side effects
```
remove this line, or add more descriptive text - what are DOM side effects?